src/meta-srv/src/failure_detector.rs

```diff
@@ -32,30 +32,27 @@ use std::collections::VecDeque;
 ///
 /// where F is the cumulative distribution function of a normal distribution with mean
 /// and standard deviation estimated from historical heartbeat inter-arrival times.
+#[derive(Clone)]
```

Comment:
The clone seems to be only used for testing. Is it possible to add a conditional compilation for it.